hw1/01-fizzbuzz.js

```diff
@@ -9,7 +9,7 @@ Use console.log() to write the proper output to the command line.
 
 **/
 
-const fizzbuzz = () => {
+
   function fizz(num) {
   for (let i = 1; i <= num; i++) {
     if (i % 2 == 0 && i % 3 == 0) {
```

Comment:
I believe the [document](https://canvas.pdx.edu/courses/58264/assignments/606549) says that divide by 3 for fizz and 5 for buzz but I think that's not a problem for the purpose of this assignment